samples/nrf9160/memfault/src/main.c

```diff
@@ -120,12 +120,24 @@ static void button_handler(uint32_t button_states, uint32_t has_changed)
 		LOG_WRN("Stack overflow will now be triggered");
 		fib(10000);
 	} else if (buttons_pressed & DK_BTN2_MSK) {
+		volatile uint32_t i;
+
+#if defined(CONFIG_BUILD_WITH_TFM)
+		/* NULL pointer exceptions are caught by TF-M, so use a
```

Comment:
No need to mention this. This looks like something that should be in release notes though